app-frontend/src/app/components/lab/diagramNodeHeader/diagramNodeHeader.html

```diff
@@ -28,8 +28,36 @@
             ng-if="$ctrl.node.apply"></span>
       {{$ctrl.typeMap[$ctrl.node.type]}}
     </div>
-    <div class="node-header-label">
-      {{$ctrl.node.metadata.label}}
+    <div>
```

Comment:
We'll need to add a class to this `div` to manage flex-box properties. Something like `node-header-name-container`.

We'll want to add a property of `flex-grow: 1` to this class as well as make it a flex-box container itself (row direction). We can then align all of the content of the div to the bottom.

Reply:
To make this look right, you'll also have to adjust the `node-header-type` class and layout. We need to prevent it from getting squished and align things.

Adding the following to the class should be sufficient for that:
```css
flex-shrink: 0;
margin-bottom: 0.25em;
height: 18px;
align-items: center;
```
After that, I'd suggest commenting out the info icon (since there is no functionality for it) and making it so that if the type lookup doesn't return anything that we display "Operation".